Templates/make_jenkins_EC2-Agent-Windows-instance.tmplt.json

```diff
@@ -0,0 +1,1613 @@
+            "Type": "String"
+        }
+    },
+    "Resources": {
+        "Ec2Res": {
+            "CreationPolicy": {
+                "ResourceSignal": {
+                    "Count": "1",
+                    "Timeout": "PT60M"
+                }
+            },
+            "Metadata": {
+                "AWS::CloudFormation::Init": {
+                    "configSets": {
+                        "appscript": [
+                            "reboot",
+                            "make-app"
+                        ],
+                        "chainscript": [
+                            "chain-script",
+                            "reboot"
+                        ],
+                        "launch": [
+                            "setup",
+                            {
+                                "Fn::If": [
+                                    "InstallCloudWatchAgent",
+                                    "install-cloudwatch-agent",
+                                    {
+                                        "Ref": "AWS::NoValue"
+                                    }
+                                ]
+                            },
+                            "watchmaker-launch",
+                            {
+                                "Fn::If": [
+                                    "ExecuteAppScript",
+                                    {
+                                        "ConfigSet": "appscript"
+                                    },
+                                    {
+                                        "Ref": "AWS::NoValue"
+                                    }
+                                ]
+                            },
+                            {
+                                "Fn::If": [
+                                    "Reboot",
+                                    "reboot",
+                                    {
+                                        "Ref": "AWS::NoValue"
+                                    }
+                                ]
+                            },
+                            {
+                              "Fn::If": [
+                                "UseChainScript",
+                                {
+                                    "ConfigSet": "chainscript"
+                                },
+                                { "Ref": "AWS::NoValue" }
+                              ]
+                            },
+                            "finalize"
+                        ],
+                        "update": [
+                            "setup",
+                            "watchmaker-update",
+                            {
+                                "Fn::If": [
```

Comment:
You've included running of the `appscript` configSet in the `Updates` section but not your `chainscript`. If you're not going to put your `chain-script` and `make-app` into the same `configSet`, this (`update`) section is incomplete: you'd also need to replicate your prior `"ConfigSet": "chainscript"` condition-block here.